tests/acceptance/standard/test_resource_groups.py

```diff
@@ -38,9 +40,59 @@ def test__update_resource_group_process_mode(self, project, add_gitlab_ci_config
                 process_mode: newest_first
             """
 
-        time.sleep(5)
         run_gitlabform(update_resource_group_config, project)
+        time.sleep(5)
 
         resource_group = project.resource_groups.get("production")
         assert resource_group.key == "production"
         assert resource_group.process_mode == "newest_first"
+
+    def test__ensure_exists_default_true(self, project, add_gitlab_ci_config, capsys):
```

Comment:
Let's remove this `add_gitlab_ci_config` fixture from all the new test cases. This fixture is scoped to function and it will execute everytime. But, the fixture uses the same `project` fixture, which is scoped to the entire class. So, running this fixture everytime is not necessary since it creates the necessary gitlab-ci in the first test case.